docs/tutorials/query-lang.md

```diff
@@ -1,13 +1,13 @@
 Write You A Query Language
 ===============================================
 
-**P.S**: *this document hasn't got up-to-date*.
+**P.S**: *this document is not up-to-date*.
 
-You may have heard of LINQ or extension methods before, and they're all embedded query languages.
+You may have heard of embedded query languages like LINQ or extension methods before.
 
-In terms of Julia ecosystem, there're already Query.jl, LightQuery.jl, DataFramesMeta.jl, etc., each of which reaches the partial or full features of a query language.
+In terms of Julia ecosystem, there is already Query.jl, LightQuery.jl, DataFramesMeta.jl, etc. These packages accomplish the partial or full features of a query language.
 
-This document is provided for you to create a concise and efficient implementation of query language, which is a way for me to exhibit the power of MLStyle.jl on AST manipulations. Additionally, I think this tutorial can be also extremely helpful to those who're developing query languages for Julia.
+This tutorial primarily shows the creation a concise and efficient query language implementation using MLStyle.jl. This demonstration illustrates the power of MLStyle.jl's ability to perform AST manipulations. Additionally, I think this tutorial can be also extremely helpful to those who're developing query languages for Julia.
```

Comment:
```suggestion
This tutorial primarily shows the creation a concise and efficient query language implemented  with MLStyle.jl. This demonstration illustrates the power of MLStyle.jl's ability to perform AST manipulations. Additionally, I think this tutorial can be also extremely helpful to those who're developing query languages for Julia.
```

Reply:
Feel like either of these could work, but lets go with yours :)